modules/RTC/TraceablePeerConnection.js

```diff
@@ -1881,7 +1882,7 @@ TraceablePeerConnection.prototype.setRemoteDescription = function(description) {
 
     // Safari WebRTC errors when no supported video codec is found in the offer.
     // To prevent the error, inject H264 into the video mLine.
-    if (browser.isSafariWithWebrtc()) {
+    if (browser.isSafariWithWebrtc() && !browser.isSafariWithVP8()) {
```

Comment:
but doesn't the new safari still have 264?

Reply:
ok, re-read the comment again and i misunderstood.  but maybe this should still be a bit different to make sure we insert something that the current safari supports?

Reply:
Chrome codecs
```
a=rtpmap:96 VP8/90000
a=rtpmap:97 rtx/90000
a=rtpmap:98 red/90000
```

Firefox codecs
```
a=rtpmap:120 VP8/90000
a=rtpmap:121 VP9/90000
a=rtpmap:126 H264/90000
```

Safari codecs
```
a=rtpmap:99 rtx/90000
a=rtpmap:96 H264/90000
a=rtpmap:100 VP8/90000
```

They at least will be using VP8